.github/workflows/docker.yml

```diff
@@ -157,6 +157,121 @@ jobs:
           to: ${{ secrets.NOTIFY_EMAIL_TO }}
           password: ${{ secrets.NOTIFY_EMAIL_PASSWORD }}
 
+  docker-a3p-upgrade-next:
+    needs: [docker-sdk, snapshot]
+    runs-on: 'depot-ubuntu-22.04-16' # ubuntu-latest
+    permissions:
+      # allow issuing OIDC tokens for this workflow run
+      id-token: write
+      # allow at least reading the repo contents, add other permissions if necessary
+      contents: read
+      # to push the resulting images
+      packages: write
+    outputs:
+      build-id: '${{ steps.bake-upgrade-next.outputs.build-id }}'
+      json-tags: '${{ steps.prefix-tags.outputs.json-tags }}'
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./.github/actions/restore-node
+        with:
+          # 18.19 causes this problem running tsx:
+          # The --loader flag was deprecated in Node v20.6.0
+          node-version: 18.18
+          # Since we can't build core eval submissions for the SDK using a newer endo,
+          # simply ignore any endo branch integration (this means we don't have full coverage)
+          ignore-endo-branch: 'true'
+
+      - name: Install deps
+        run: |
+          # Enable corepack for packageManager config
+          corepack enable || sudo corepack enable
+          yarn install
+        working-directory: a3p-integration
+
+      - name: Save SDK_TAG
+        run: echo "SDK_TAG=${{ needs.snapshot.outputs.tag }}" >> $GITHUB_ENV
+
+      - name: Override the buildConfig variables
+        run: |
+          jq --arg PLATFORMS "$DOCKER_PLATFORMS" \
+            '. * {"agoricSyntheticChain": {
+              "platforms": ($PLATFORMS | split(","))
+            }}' package.json > ./package.json.new
+          mv ./package.json.new ./package.json
+
+          jq --arg SDK_TAG "$SDK_TAG" \
+            '. * {"agoricProposal": {
+              "sdkImageTag": $SDK_TAG,
+            }}' proposals/n\:upgrade-next/package.json > ./package.json.new
+          mv ./package.json.new proposals/n\:upgrade-next/package.json
+        working-directory: a3p-integration
+
+      - name: Prefix tags
+        id: prefix-tags
+        run: |
+          # These tags don't have an image name attached yet and are JSONified
+          # with newline separators because they are used as input to
+          # docker/metadata-action.
+          PREFIX=a3p-use-upgrade-next-
```

Comment:
this image needs to be separate from agoric-sdk. We've already had people burn effort thinking `agoric-sdk:a3p-use-upgrade-next-20240911203722-bc66a5` gives you an agoric-sdk. 

some alternatives: 
-  https://ghrc.io/agoric/agoric-3-drafts 
-  https://ghrc.io/agoric/a3p-integration

I like the latter because the relationship with agoric-sdk is clear

Reply:
The latter is fine.